### ISSUE TITLE
Handling Municipality with DWC Import in TaxonWorks

### ISSUE THREAD

**Issue by brennaldecker** [SpeciesFileGroup/taxonworks#1918]
**Describe Issue**

In DWC field, municipality is an accepted field, but TaxonWorks does not have this component. 

~ How should we handle municipalities? Should they:

1. Be included in the locality field?
2. Have a separate field built into TaxonWorks that matches DWC terms?
- This will help in creating a completed Collection Event (see screenshot below in Expected behavior)

**To Reproduce**
1. When I fill out the datasheet, I separate the municipality (Bishop Hills) from locality (Bluff on W Amarillo Creek)
<img width="1216" alt="Screen Shot 2020-12-02 at 9 55 42 AM" src="https://user-images.githubusercontent.com/75384782/100905184-fffc8a00-3484-11eb-8024-a1cbf76e118e.png">

2. Then I upload using DWC-A Importer on Sandworm (INHS DWC Testing)

3. And I make sure it uploaded properly, and go to one of the records, click the stacked icon in the top right corner to get to the Comprehensive specimen digitization page, and check both the locality and verbatim Label fields...

<img width="878" alt="Screen Shot 2020-12-02 at 9 56 02 AM" src="https://user-images.githubusercontent.com/75384782/100905209-0a1e8880-3485-11eb-8ae9-c4b16066b6a5.png">

**Expected behavior**

1. The verbatim Label should have the information from the DWC verbatimLabel field, but that does not get populated.
2. In the Collecting Event field, it does not include the municipality, even though it is part of the verbatimLabel DWC field.
<img width="1004" alt="Screen Shot 2020-12-02 at 10 09 50 AM" src="https://user-images.githubusercontent.com/75384782/100906572-89608c00-3486-11eb-8229-d90a62e6b6e6.png">

3. It would be great to see municipality as a separate field that is different from a specific locality. This may be important for biogeography analysis or understanding the distribution of sampling effort within a state/province.

**Environment (please identify where you experience this bug:**
- Sandworm -  https://sandworm.taxonworks.org - INHS
- FireFox on Mac

**Comment by mjy** [comment]
Thanks *very * much for the testing @brennaldecker.

Question. Is municipality a DwC Field? (Ping @debpaul )

**Comment by brennaldecker** [comment]
@mjy Yes, municipalities is a DWC field = municipality (http://rs.tdwg.org/dwc/terms/municipality)

**Comment by mjy** [comment]
@LocoDelAssembly We'll need to map it to custom data attribute with the municipality IRI as discussed in the meeting this morning.  Likely create or match on demand?

**Comment by mjy** [comment]
@brennaldecker steps to capture (will export with new export coming)
* Add a Predicate in the "Manage controlled vocabulary terms" 
* Give it a name and defintion, but most importantly use `http://rs.tdwg.org/dwc/terms/municipality` (double checking not the IRI version) in the URL field.
* GOTO project->preferences (or have your project admin do it)
* Click "Collecting Event"
* Select your Predicate (is autosave)

At this point you will see a Municipality field in Custom attributes sections

**Comment by mjy** [comment]
@debpaul I believe we can close this thanks to your work with Jared and Hernán?

**Comment by LocoDelAssembly** [comment]
Currently it works as data attribute, no special handling, just the default behavior of the importer checking if there is a DwC term enabled in the project as data attribute for CO and/or CE matching the one in the dataset. No native import is performed, and I think `municipality` is not exported either (although the field itself does exist in `dwc_occurrences` table).
